### ISSUE TITLE
Automatic sizing sometimes goes beyond bounds

### ISSUE THREAD

**Issue by AngledLuffa** [almende/vis#578]
Here is a simple, possibly silly example where I would expect automatic sizing to fit the graph in the space provided, but the top and bottoms of a couple nodes are cut off.  If this is expected, please feel free to disregard, but I thought it would be able to fit this many nodes in its display area.

```
<html>
<head>
    <style type="text/css">
        #mynetwork {
            height: 500px;
            border: 1px solid lightgray;
        }
    </style>
</head>

<body>
    <div id="mynetwork"></div>

    <link href="http://visjs.org/dist/vis.css" rel="stylesheet" type="text/css" />
    <script type="text/javascript" src="http://visjs.org/dist/vis.js"></script>

    <script type="text/javascript">
        function draw() {
            var nodes = new vis.DataSet();
            var edges = new vis.DataSet();

            nodes.add({
                id: 0,
                label: "foo",
                level: 0,
                fontSize: 24,
                color: {
                    background: "#F88888",
                    border: "#884848",
                }
            });

            nodes.add({
                id: 1,
                label: "foo",
                level: 1,
                fontSize: 24,
                color: {
                    background: "#F88888",
                    border: "#884848",
                }
            });

            nodes.add({
                id: 2,
                label: "foo",
                level: 2,
                fontSize: 24,
                color: {
                    background: "#F88888",
                    border: "#884848",
                }
            });

            nodes.add({
                id: 3,
                label: "foo",
                level: 3,
                fontSize: 24,
                color: {
                    background: "#F88888",
                    border: "#884848",
                }
            });

            edges.add({
               from: 0,
               to: 1
            });
                
            edges.add({
               from: 1,
               to: 2
            });
                
            edges.add({
               from: 2,
               to: 3
            });
                

            // create a network
            var container = document.getElementById('mynetwork');
            var data = {
                nodes: nodes,
                edges: edges
            };

            var options = {
                hierarchicalLayout: {
                    direction: "UD"
                },
                edges: {style:"arrow"},
                smoothCurves:false
            };
            network = new vis.Network(container, data, options);
        }

        draw();
    </script>

</body>
</html>
```

**Comment by AlexDM0** [comment]
Hi,

You're right, that should work. I'll look into it  

Regards,

Alex

**Comment by AlexDM0** [comment]
Hi, 

This is fixed in develop, from issue #575. Thank you for reporting this.

Regards,

Alex
